custom/icds_reports/tasks.py

```diff
@@ -1221,14 +1221,29 @@ def aggregate_validation_helper():
                                                                        'wer_weighed', 'wer_eligible',
```

Comment:
These need to be filtered by date as well

Reply:
added the filter